Exec Doc 2025-01-23 (#2827)
makerdao/community: governance/votes/Executive vote - January 23, 2025.md

---

> title: Template - [Executive Vote] Stability Fee Changes, DAI Savings Rate Changes, Launch Project Funding, Integration Boost Funding, Aligned Delegate Compensation, Atlas Core Developer Payments, New Suckable USDS Vest Contract Setup, New Suckable Transferrable SKY Contract Setup, Setting Facilitator Payment Streams, SparkLend-Aave Q4 2024 Revenue Share Payment, TACO DAO Resolution Approval, Spark Proxy Spell - January 23, 2025

Reviewer: ```suggestion
title: Template - [Executive Vote] Stability Fee Changes, DAI Savings Rate Changes, Launch Project Funding, Integration Boost Funding, Aligned Delegate Compensation, Atlas Core Developer Payments, New Suckable USDS Vest Contract Setup, New Transferrable SKY Vest Contract Setup, Setting Facilitator Payment Streams, SparkLend-Aave Q4 2024 Revenue Share Payment, TACO DAO Resolution Approval, Spark Proxy Spell - January 23, 2025
```

> summary: Stability Fee changes for multiple vault types, DAI Savings Rate decrease, Launch Project and Integration Boost funding, Aligned Delegate compensation for December 2024, Atlas Core Developer payments for December 2024, New Suckable USDS Vest Contract Setup, New Suckable Transferrable SKY Contract Setup, Setting Facilitator Payment Streams, SparkLend-Aave Q4 2024 Revenue Share Payment, TACO DAO Resolution Approval, Spark Proxy Spell.

Reviewer 2: ```suggestion
summary: Stability Fee changes for ALLOCATOR-SPARK-A, DAI Savings Rate decrease, Launch Project and Integration Boost funding, Aligned Delegate compensation for December 2024, Atlas Core Developer payments for December 2024, New Suckable USDS Vest Contract setup, New Suckable Transferrable SKY Contract setup, Setting Facilitator Payment Streams, SparkLend-Aave Q4 2024 Revenue Share Payment, TACO DAO Resolution approval, Spark Proxy Spell.
```

Reviewer: additional suggestions to those of @CivicSage3 

```suggestion
summary: Stability Fee change for ALLOCATOR-SPARK-A, DAI Savings Rate decrease, Launch Project and Integration Boost funding, Aligned Delegate compensation for December 2024, Atlas Core Developer payments for January 2025, New Suckable USDS Vest Contract setup, New  Transferrable SKY Vest Contract setup, Setting Facilitator Payment Streams, SparkLend-Aave Q4 2024 Revenue Share Payment, TACO DAO Resolution approval, Spark Proxy Spell.
```

date: 2025-01-23T00:00:00.000Z
address: "$spell_address"

---

> # [Executive Proposal] Stability Fee Changes, DAI Savings Rate Changes, Launch Project Funding, Integration Boost Funding, Aligned Delegate Compensation, Atlas Core Developer Payments, New Suckable USDS Vest Contract Setup, New Suckable Transferrable SKY Contract Setup, Setting Facilitator Payment Streams, SparkLend-Aave Q4 2024 Revenue Share Payment, TACO DAO Resolution Approval, Spark Proxy Spell - January 23, 2025

Reviewer: ```suggestion
# [Executive Proposal] Stability Fee Changes, DAI Savings Rate Changes, Launch Project Funding, Integration Boost Funding, Aligned Delegate Compensation, Atlas Core Developer Payments, New Suckable USDS Vest Contract Setup, New Transferrable SKY Vest Contract Setup, Setting Facilitator Payment Streams, SparkLend-Aave Q4 2024 Revenue Share Payment, TACO DAO Resolution Approval, Spark Proxy Spell - January 23, 2025
```

The Governance Facilitators, Sidestream, and Dewiz have placed an executive proposal into the voting system. MKR/SKY Holders should vote for this proposal if they support the following alterations to the Sky Protocol.

If you are new to voting in the Sky Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

---

## Executive Summary

If this executive proposal passes, the following **actions** will occur within the Sky Protocol:

> - The [Stability Fees](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.3_Stability_Fee-67e40a3b-f1c2-4dc6-b502-2affeab0b232|57eaf45219bea3b430c2) for multiple vault types will be decreased.

Reviewer: additional change to @CivicSage3 suggestion

```suggestion
- The [Stability Fee](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.3_Stability_Fee-67e40a3b-f1c2-4dc6-b502-2affeab0b232|57eaf45219bea3b430c2) for ALLOCATOR-SPARK-A will be decreased.
```

- A total of **314,567 DAI** will be distributed as SparkLend-Aave Revenue Share payment for Q4 2024.
- A TACO DAO Resolution will be approved.
- A Spark proxy spell will be executed.

**Voting for this executive proposal will place your MKR/SKY in support of the actions outlined above.**

Unless otherwise noted, the actions listed above are subject to the [GSM Pause Delay](https://sky-atlas.powerhouse.io/#A.1.8.2.1_Pause_Delay-a98b8227-95f6-4711-9d8d-f52cbc6ad2d0|0db30758e055). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Sky Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to [**30 hours**](https://sky-atlas.powerhouse.io/#A.1.8.2.1.2_Pause_Delay_Current_Value-09d2514b-3169-4755-a654-2c774456980d|0db30758e055d2d0).

This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**.

If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Sky Protocol.

---

## Proposal Details

> ### Stability Fee Changes

Reviewer: ```suggestion
### Stability Fee Change
```

- **Authorization**: [$link_to_approval](TBC)
- **Proposal**: [$link_to_proposal](TBC)

If this executive proposal passes, then the [Stability Fees (SF)](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.3_Stability_Fee-67e40a3b-f1c2-4dc6-b502-2affeab0b232|57eaf45219bea3b430c2) for the vault types listed below will be changed as follows:

- Decrease ETH-A SF by TBC percentage points from 19.75% to **TBC%**.
- Decrease ETH-B SF by TBC percentage points from 20.25% to **TBC%**.
- Decrease ETH-C SF by TBC percentage points from 19.50% to **TBC%**.
- Decrease WSTETH-A SF by TBC percentage points from 20.75% to **TBC%**.
- Decrease WSTETH-B SF by TBC percentage points from 20.50% to **TBC%**.
- Decrease WBTC-A SF by TBC percentage points from 24.25% to **TBC%**.

> - Decrease WBTC-B SF by TBC percentage points from 24.75% to **TBC%**.

Reviewer 2: ```suggestion
```

> - Decrease WBTC-C SF by TBC percentage points from 24.00% to **TBC%**.

Reviewer 2: ```suggestion
```

> - Decrease ALLOCATOR-SPARK-A SF by TBC percentage points from 18.13% to **TBC%**.

Reviewer 2: ```suggestion
- Decrease ALLOCATOR-SPARK-A SF by 6.88 percentage points from 12.25% to **5.37%**.
```

> ### DAI Savings Rate Changes

Reviewer: ```suggestion
### DAI Savings Rate Reduction
```

> - **Authorization**: [$link_to_approval](TBC)

Reviewer 2: ```suggestion
- **Authorization**: [Ecosystem Approval](https://forum.sky.money/t/stability-scope-parameter-changes-20-spark-liquidity-layer-dsr/25861/2)
```

> - **Proposal**: [$link_to_proposal](TBC)

Reviewer 2: ```suggestion
- **Proposal**: [Forum post](https://forum.sky.money/t/stability-scope-parameter-changes-20-spark-liquidity-layer-dsr/25861)
```

- Transfer **24 million SKY** to the Launch Project at [0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F](https://etherscan.io/address/0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F).

### Integration Boost Funding

- **Authorization**: [Atlas A.5.2.1.2](https://sky-atlas.powerhouse.io/A.5.2.1.2_Integration_Boost/129f2ff0-8d73-8057-850b-d32304e9c91a%7C8d5a9e88cf49)
- **Proposal**: [Forum Post](https://forum.sky.money/t/utilization-of-the-integration-boost-budget-a-5-2-1-2/25536/4)

If this executive proposal passes, then the following Integration Boost funding distribution will be made:

- Transfer **2.5 million USDS** to [0xD6891d1DFFDA6B0B1aF3524018a1eE2E608785F7](https://etherscan.io/address/0xD6891d1DFFDA6B0B1aF3524018a1eE2E608785F7).

### Aligned Delegate Compensation for December 2024

- **Authorization**: [Atlas A.1.5.8](https://sky-atlas.powerhouse.io/A.1.5.8_Budget_For_Prime_Delegate_Slots/e3e420fc-9b1f-4fdc-9983-fcebc45dd3aa%7C0db3af4ece0c)
- **Proposal**: [Forum Post](https://forum.sky.money/t/december-2024-aligned-delegate-compensation/25854)

> If this executive proposal passes, then a total of **25,402 USDS** will be distributed to sevenix Aligned Delegates and their teams as compensation for December 2024.

Reviewer 3: ```suggestion
If this executive proposal passes, then a total of **25,402 USDS** will be distributed to seven Aligned Delegates as compensation for December 2024.
```

> If this executive proposal passes, then a total of **25,402 USDS** will be distributed to sevenix Aligned Delegates and their teams as compensation for October 2024.

Reviewer 4: ```suggestion
If this executive proposal passes, then a total of **25,402 USDS** will be distributed to sevenix Aligned Delegates and their teams as compensation for December 2024.
```

- **Proposal**: [Forum Post](https://forum.sky.money/t/atlas-core-development-payment-requests-january-2025/25842)

If this executive proposal passes, then a total of **122,438 USDS** and **988,000 SKY** will be distributed to two Atlas Core Development contributors and their teams:

| Contributor     | USDS Amount | SKY Amount  | Address                                                                                                               |
|-----------------|-------------|-------------|-----------------------------------------------------------------------------------------------------------------------|
| BLUE            | 83,602 USDS | 550,000 SKY | [0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf](https://etherscan.io/address/0xb6C09680D822F162449cdFB8248a7D3FC26Ec9Bf) |
| Kohla (Cloaky)  | 10,000 USDS | 0 SKY       | [0x73dFC091Ad77c03F2809204fCF03C0b9dccf8c7a](https://etherscan.io/address/0x73dFC091Ad77c03F2809204fCF03C0b9dccf8c7a) |
| Ennoia (Cloaky) | 10,000 USDS | 0 SKY       | [0xA7364a1738D0bB7D1911318Ca3FB3779A8A58D7b](https://etherscan.io/address/0xA7364a1738D0bB7D1911318Ca3FB3779A8A58D7b) |

> | Cloaky          | 18,836 USDS | 438,000 SKY | [0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818](https://etherscan.io/address/0x869b6d5d8FA7f4FFdaCA4D23FFE0735c5eD1F818) |

Reviewer: Not blocking but consider splitting tables so there is a USDS table and a SKY table. This will match the code and sheet more accurately.

### New Suckable USDS Vest Contract Setup

- **Authorization**: [Ecosystem Authorization](https://forum.sky.money/t/proposed-housekeeping-items-upcoming-executive-spell-2025-01-23/25852/2)
- **Proposal**: [Forum Post](https://forum.sky.money/t/proposed-housekeeping-items-upcoming-executive-spell-2025-01-23/25852)

If this executive proposal passes, then a new suckable USDS vest contract will be set up through the following actions:

- Authorize the new USDS vest contract ([0xc447a9745aDe9A44Bb9E37B7F6C92f9582544110](https://etherscan.io/address/0xc447a9745aDe9A44Bb9E37B7F6C92f9582544110)) to access the surplus buffer (MCD_VAT).

> - Set the maximum vesting speed (`cap`) on the new USDS vest contract to **46,200 USDS** per 30 days.

Reviewer: ```suggestion
- Set the maximum vesting speed (`cap`) on the new USDS vest contract to **46,200 USDS per 30 days**.
```

- Add the new USDS vest contract ([0xc447a9745aDe9A44Bb9E37B7F6C92f9582544110](https://etherscan.io/address/0xc447a9745aDe9A44Bb9E37B7F6C92f9582544110)) to Chainlog under `MCD_VEST_USDS`.

### New Transferrable SKY Vest Contract Setup

- **Authorization**: [Ecosystem Authorization](https://forum.sky.money/t/proposed-housekeeping-items-upcoming-executive-spell-2025-01-23/25852/2)
- **Proposal**: [Forum Post](https://forum.sky.money/t/proposed-housekeeping-items-upcoming-executive-spell-2025-01-23/25852)

If this executive proposal passes, then a new transferrable SKY vest contract will be set up through the following actions:

- Convert **624 MKR** held in the Pause Proxy to SKY via the `MKR_SKY` contract.

> - Approve the new SKY vest contract ([0x67eaDb3288cceDe034cE95b0511DCc65cf630bB6](https://etherscan.io/address/0x67eaDb3288cceDe034cE95b0511DCc65cf630bB6)) to take a total of **14,968,800 SKY** from the treasury (`MCD_PAUSE_PROXY`.)

Reviewer 4: ```suggestion
- Approve the new SKY vest contract ([0x67eaDb3288cceDe034cE95b0511DCc65cf630bB6](https://etherscan.io/address/0x67eaDb3288cceDe034cE95b0511DCc65cf630bB6)) to take a total of **14,256,000 SKY** from the treasury (`MCD_PAUSE_PROXY`.)
```

> - Set the maximum vesting speed (`cap`) on the new SKY vest contract to **475,200 SKY** per 30 days.

Reviewer: ```suggestion
- Set the maximum vesting speed (`cap`) on the new SKY vest contract to **475,200 SKY per 30 days**.
```

|--------------|-----------------------------------------------------------------------------------------------------------------------|--------------|------------|------------|
| Endgame Edge | [0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1](https://etherscan.io/address/0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1) | 462,000      | 2025-02-01 | 2025-12-31 |
| JanSky       | [0xf3F868534FAD48EF5a228Fe78669cf242745a755](https://etherscan.io/address/0xf3F868534FAD48EF5a228Fe78669cf242745a755) | 462,000      | 2025-02-01 | 2025-12-31 |
| Ecosystem    | [0xFCa6e196c2ad557E64D9397e283C2AFe57344b75](https://etherscan.io/address/0xFCa6e196c2ad557E64D9397e283C2AFe57344b75) | 462,000      | 2025-02-01 | 2025-12-31 |

#### SKY Streams

| Facilitator  | Address                                                                                                               | Amount (SKY) | Start Date | End Date   |
|--------------|-----------------------------------------------------------------------------------------------------------------------|--------------|------------|------------|

> | Endgame Edge | [0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1](https://etherscan.io/address/0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1) | 4,752 SKY    | 2025-02-01 | 2025-12-31 |

Reviewer 4: ```suggestion
| Endgame Edge | [0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1](https://etherscan.io/address/0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1) | 4,752,000 SKY    | 2025-02-01 | 2025-12-31 |
```

> | JanSky       | [0xf3F868534FAD48EF5a228Fe78669cf242745a755](https://etherscan.io/address/0xf3F868534FAD48EF5a228Fe78669cf242745a755) | 4,752 SKY    | 2025-02-01 | 2025-12-31 |

Reviewer 4: ```suggestion
| JanSky       | [0xf3F868534FAD48EF5a228Fe78669cf242745a755](https://etherscan.io/address/0xf3F868534FAD48EF5a228Fe78669cf242745a755) | 4,752,000 SKY    | 2025-02-01 | 2025-12-31 |
```

> | Ecosystem    | [0xFCa6e196c2ad557E64D9397e283C2AFe57344b75](https://etherscan.io/address/0xFCa6e196c2ad557E64D9397e283C2AFe57344b75) | 4,752 SKY    | 2025-02-01 | 2025-12-31 |

Reviewer 4: ```suggestion
| Ecosystem    | [0xFCa6e196c2ad557E64D9397e283C2AFe57344b75](https://etherscan.io/address/0xFCa6e196c2ad557E64D9397e283C2AFe57344b75) | 4,752,000 SKY    | 2025-02-01 | 2025-12-31 |
```

### SparkLend-Aave Q4 2024 Revenue Share Payment

- **Authorization**: [Ecosystem Authorization](https://forum.sky.money/t/spark-aave-revenue-share-calculations-payments-6-q4-2024/25820/2)
- **Proposal**: [Forum Post](https://forum.sky.money/t/spark-aave-revenue-share-calculations-payments-6-q4-2024/25820)

If this executive proposal passes, then a payment of **314,567 DAI** will be made to a smart contract controlled by Aave Governance at [0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c](https://etherscan.io/address/0x464C71f6c2F760DdA6093dCB91C24c39e5d6e18c) as part of the SparkLend-Aave Revenue Share.

### TACO DAO Resolution Approval

- **Authorization:** [Forum post](https://forum.sky.money/t/spark-aave-revenue-share-calculations-payments-6-q4-2024/25820/2)
- **Proposal:** [Forum post](https://forum.sky.money/t/project-andromeda-risk-legal-assessment/20969/12)

> If this executive proposal passes, the resolutions posted within IPFC hash [bafkreieqcricvrwb7ndxep6unlhhm6iie3dlkr3cl4tdypinjws4pycalq](https://gateway.pinata.cloud/ipfs/bafkreieqcricvrwb7ndxep6unlhhm6iie3dlkr3cl4tdypinjws4pycalq) will be approved.

Reviewer: ```suggestion
If this executive proposal passes, the resolutions posted within IPFS hash [bafkreieqcricvrwb7ndxep6unlhhm6iie3dlkr3cl4tdypinjws4pycalq](https://gateway.pinata.cloud/ipfs/bafkreieqcricvrwb7ndxep6unlhhm6iie3dlkr3cl4tdypinjws4pycalq) will be approved.
```

### Spark Proxy Spell

- **Authorization**: Governance Polls [1180](https://vote.makerdao.com/polling/QmRAavx5), [1181](https://vote.makerdao.com/polling/QmY4D1u8), [1182](https://vote.makerdao.com/polling/QmU3Xu4W), [1183](https://vote.makerdao.com/polling/QmexceBK)
- **Proposal**: [Forum Posts](https://forum.sky.money/t/jan-23-2025-proposed-changes-to-spark-for-upcoming-spell/25825)

> If this executive proposal passes, then a Spark Proxy Spell will be executed at [TBC](https://etherscan.io/address/TBC) with the following contents:

Reviewer 2: ```suggestion
If this executive proposal passes, then a Spark Proxy Spell will be executed at [0xFe447da54AdD21a8503eb81d328c5D60fE90eC26](https://etherscan.io/address/0xFe447da54AdD21a8503eb81d328c5D60fE90eC26) with the following contents:
```

### New Transferrable SKY Vest Contract Setup

- **Authorization**: [Ecosystem Authorization](https://forum.sky.money/t/proposed-housekeeping-items-upcoming-executive-spell-2025-01-23/25852/2)
- **Proposal**: [Forum Post](https://forum.sky.money/t/proposed-housekeeping-items-upcoming-executive-spell-2025-01-23/25852)

If this executive proposal passes, then a new transferrable SKY vest contract will be set up through the following actions:

- Convert **624 MKR** held in the Pause Proxy to SKY via the `MKR_SKY` contract.
- Approve the new SKY vest contract ([0x67eaDb3288cceDe034cE95b0511DCc65cf630bB6](https://etherscan.io/address/0x67eaDb3288cceDe034cE95b0511DCc65cf630bB6)) to take a total of **14,256,000 SKY** from the treasury (`MCD_PAUSE_PROXY`.)
- Set the maximum vesting speed (`cap`) on the new SKY vest contract to **475,200 SKY per 30 days**.
- Add the new SKY vest contract ([0x67eaDb3288cceDe034cE95b0511DCc65cf630bB6](https://etherscan.io/address/0x67eaDb3288cceDe034cE95b0511DCc65cf630bB6)) to Chainlog under `MCD_VEST_SKY_TREASURY`.

### Facilitator Payment Streams Setup

> - **Authorization**: [Atlas A.1.6.2.4.1](https://sky-atlas.powerhouse.io/A.1.6.2.4.1_List_of_Facilitator_Budgets/c511460d-53df-47e9-a4a5-2e48a533315b%7C0db3343515519c4a)

Reviewer 3: ```suggestion
- **Authorization**: [Atlas A.1.6.2.4.1](https://sky-atlas.powerhouse.io/A.1.6.2.4.1_List_of_Facilitator_Budgets/c511460d-53df-47e9-a4a5-2e48a533315b%7C0db3343515519c4a)
- **Proposal**: [Forum Post](https://forum.sky.money/t/proposed-housekeeping-items-upcoming-executive-spell-2025-01-23/25852)
```

  - Liquidation Protocol Fee: **10%**
  - [High Efficiency Mode (`Emode`)](https://sky-atlas.powerhouse.io/#A.3.8.1.5.1.6_High_Efficiency_Mode_Category_Definition-09de838a-b46c-4cdb-9467-adacdbcf4c00|57eaf45219be608847d6) Category: **0**

#### [Mainnet Liquidity Layer] Onboard Aave Prime USDS

- Onboarding Aave Prime USDS onto the SparkLend Liquidity Layer with the following parameters:
  - Deposits:
    - Max amount: **50 million USDS**
    - Slope: **50 million USDS per day**
  - Withdrawals
    - Max amount: **Unlimited**
    - Slope: **Unlimited**

#### [Mainnet Liquidity Layer] Onboard SparkLend USDS, USDC

- Onboarding SparkLend USDS and SparkLend USDC onto the SparkLend Liquidity Layer with the following parameters:
  - **USDS**:
    - Deposits:
      - Max amount: **150 million USDS**
      - Slope: **75 million USDS per day**
    - Withdrawals:
      - Max amount: **Unlimited**
      - Slope: **Unlimited**
  - **USDC**:
    - Deposits:
      - Max amount: **20 million USDC**
      - Slope: **10 million USDC per day**
    - Withdrawals:
      - Max amount: **Unlimited**
      - Slope: **Unlimited**

#### [Mainnet Liquidity Layer] Modify Rate Limits

> - Setting the following rate limit parameters for Spark Liquidity Layer Mainnet USDS and USDC:

Reviewer 3: ```suggestion
- Setting the following [rate limit parameters](https://docs.spark.fi/dev/spark-liquidity-layer/spark-alm-controller#rate-limits) for Spark Liquidity Layer Mainnet USDS and USDC:
```

date: 2025-01-23T00:00:00.000Z
address: "$spell_address"

---
# [Executive Proposal] Stability Fee Change, DAI Savings Rate Change, Launch Project Funding, Integration Boost Funding, Aligned Delegate Compensation, Atlas Core Developer Payments, New Suckable USDS Vest Contract Setup, New Transferrable SKY Vest Contract Setup, Setting Facilitator Payment Streams, SparkLend-Aave Q4 2024 Revenue Share Payment, TACO DAO Resolution Approval, Spark Proxy Spell - January 23, 2025

The Governance Facilitators, Sidestream, and Dewiz have placed an executive proposal into the voting system. MKR/SKY Holders should vote for this proposal if they support the following alterations to the Sky Protocol.

If you are new to voting in the Sky Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.

---

## Executive Summary

If this executive proposal passes, the following **actions** will occur within the Sky Protocol:

- The [Stability Fee](https://sky-atlas.powerhouse.io/#A.3.8.1.1.2.3_Stability_Fee-67e40a3b-f1c2-4dc6-b502-2affeab0b232|57eaf45219bea3b430c2) for ALLOCATOR-SPARK-A will be decreased.

> - The [DAI Savings Rate (DSR](https://sky-atlas.powerhouse.io/A.4.3.1_Savings_Rates/5586fe33-b0af-41f0-9629-b1716c7c0967%7Cb3417d54) will be decreased.

Reviewer 5: ```suggestion
- The [DAI Savings Rate (DSR)](https://sky-atlas.powerhouse.io/A.4.3.1_Savings_Rates/5586fe33-b0af-41f0-9629-b1716c7c0967%7Cb3417d54) will be decreased.
```
